### ISSUE TITLE
[tado] Handler disposed problems

### ISSUE THREAD

**Issue by bxdavies** [openhab/openhab-addons#6670]
## Introduction
I use the tado binding to connect to my tado system but I'm getting the following error's repeatedly:

### System
Raspberry 3B+Running Openhabian 2.5.0-1
## Logs
```
2019-12-23 10:04:56.593 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:hotwater tried accessing its bridge although the handler was already disposed.
2019-12-23 10:04:56.714 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:hallheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:04:56.836 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:bensbedroomheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:04:56.916 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:mattandwendysbedroomheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:04:56.924 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:bathroomheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:04:56.949 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:loungeheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:04:56.993 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:nevesbedroomheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:04:57.019 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:commonheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:05:26.602 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:hotwater tried accessing its bridge although the handler was already disposed.
2019-12-23 10:05:26.717 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:hallheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:05:26.840 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:bensbedroomheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:05:26.918 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:mattandwendysbedroomheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:05:26.928 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:bathroomheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:05:26.951 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:loungeheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:05:26.995 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:nevesbedroomheating tried accessing its bridge although the handler was already disposed.
2019-12-23 10:05:27.023 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler TadoZoneHandler of thing tado:zone:house:commonheating tried accessing its bridge although the handler was already disposed.
```

## Potential fixes
So doing a quick google search you find [this](https://community.openhab.org/t/handler-disposed/49168) which is the same problem but with chrome cast it then gets fixed in #3813 [here](https://github.com/openhab/openhab-addons/pull/3813)

So I think the Tado binding may have the same problem as the Chromecast binding 

Do you agree with conclusion? Or is it a different issue?

**Comment by Mickroz** [comment]
im seeing the same, after i deleted the thing and removed the device from tado.

**Comment by andrewfg** [comment]
Closed by #10550
